### PR TITLE
feat(components-native): Import Text to components-native

### PR DESCRIPTION
**Pull request by taylorvnoj**
## Motivations

Import `Text` to `components-native`

### Added

 
- `Text` RN component now exists
- test & snapshot files also added

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/Text/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { Text } from "./Text";
```
Comment (rodrigoeidelvein) [inline packages/components-native/src/Text/index.ts:1]:
```suggestion
export { Text, TextLevel } from "./Text";
```

There's a place in jobber-mobile where it imports TextLevel, we should export it too.

**Review by rodrigoeidelvein — CHANGES_REQUESTED** [review]
Just a small thing

**Review by rodrigoeidelvein — CHANGES_REQUESTED** [review]
Just a small thing

File: packages/components-native/src/Text/index.ts (hunk 1 above)
Comment (taylorvnoj) [reply packages/components-native/src/Text/index.ts:1]: Great call out!

**Review by rodrigoeidelvein — APPROVED** [review]
LGTM!

**Pull request merged**